src/routes/tarin-routes.js

```diff
@@ -0,0 +1,7 @@
+const tarinController = require('../controllers/tarin-controller');
+
+module.exports = (app) => {
+    app.get('/init', tarinController.tarinList)
```

Comment:
why is this route called /init? having the route named with the resoure is much more preferable in my opinion 
what do you think?
something like GET /tarin for gettting list of all tarins ?